geps/gep-1709.md

```diff
@@ -12,32 +12,24 @@ the Gateway API project and receive recognition (e.g. badges).
 
 ## TODO
 
-The following are items that we intend to resolve before we consider this GEP
-`implementable`:
-
-- We still need to sort out how we're going to display conformance results.
```

Comment:
Check the commit history, it got moved down to "required to move past experimental" rather than "required to consider implementable. The thinking here is that the machinery we have provides a surface well enough for practically any display layer to be built on top, so it doesn't need to block (in fact, doesn't need to block GA) that we have a fancy display layer.